### PR TITLE
Prettify diff

### PR DESCRIPTION
**Pull request by SHyx0rmZ**
No longer reverse the diff (I would be very happy if somebody can explain why we did this) and also make sure that the first line in the diff has the same indentation as the others.

### CODE REVIEW

File: src/Commands/DiffCommand.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ private function printJobDiff($jobName)
         $jobDiff = $jobComparisonBusinessCase->getJobDiff($jobName);
 
         foreach ($jobDiff as $property => $diff) {
-            $diffLines = array_reverse(explode(PHP_EOL, $diff));
+            $diffLines = explode(PHP_EOL, $diff);
```
Comment (ppar) [inline src/Commands/DiffCommand.php]: Is this supposed to have a visible effect? I don't see any difference in the output of "chapi diff" from this branch compared to what master does.

Comment (SHyx0rmZ) [inline src/Commands/DiffCommand.php]: It is. It should switch the line removing and adding a change, for example. It also should switch the opening and closing braces for a property.

**Comment by andygrunwald** [comment]
Can you post the result before and the result after applying the patch?

**Comment by SHyx0rmZ** [comment]
Before the patch:

```
diff /foo-bar
  container:   }
  container:       ]
  container:           }
  container:               "mode": "RO"
  container:               "hostPath": "/foo",
  container:               "containerPath": "/foo",
  container:           {
  container:           },
  container:               "mode": "RW"
  container:               "hostPath": "/bar",
  container:               "containerPath": "/bar",
  container:           {
  container:       "volumes": [
  container:       },
  container:           "parameters": []
  container:           "forcePullImage": true,
  container:           "privileged": false,
  container:           ],
  container:               }
  container:                   "name": null
  container:                   "protocol": "tcp",
+ container:                  "servicePort": 0,
- container:                  "servicePort": 10000,
  container:                   "hostPort": 0,
  container:                   "containerPort": 80,
  container:               {
  container:           "portMappings": [
  container:           "network": "BRIDGE",
  container:           "image": "foo/bar:latest",
  container:       "docker": {
  container:       "type": "DOCKER",
  container: {
```

After the patch:

```
diff /foo-bar
  container:   {
  container:       "type": "DOCKER",
  container:       "docker": {
  container:           "image": "foo/bar:latest",
  container:           "network": "BRIDGE",
  container:           "portMappings": [
  container:               {
  container:                   "containerPort": 80,
  container:                   "hostPort": 0,
- container:                   "servicePort": 10000,
+ container:                   "servicePort": 0,
  container:                   "protocol": "tcp",
  container:                   "name": null
  container:               }
  container:           ],
  container:           "privileged": false,
  container:           "forcePullImage": true,
  container:           "parameters": []
  container:       },
  container:       "volumes": [
  container:           {
  container:               "containerPath": "/bar",
  container:               "hostPath": "/bar",
  container:               "mode": "RW"
  container:           },
  container:           {
  container:               "containerPath": "/foo",
  container:               "hostPath": "/foo",
  container:               "mode": "RO"
  container:           }
  container:       ]
  container:   }
```

Have a look at the braces and brackets before applying the patch.

**Pull request merged**